README.md

```diff
@@ -1,2 +1,83 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or client) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (eg. developers) have performed the specified tasks and there haven't been any malicious changes in the files.
```

Comment:
I think we could avoid the use of layout without explaining what it is. Overall this feels ok though.